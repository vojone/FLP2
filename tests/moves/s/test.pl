:- consult("../../../cube").

:- read_cube(C, _), move_s_cw(C, _, _, RC), write_cube(RC).
