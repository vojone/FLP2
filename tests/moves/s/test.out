123
443
789
444 141 222 333
111 252 333 424
111 262 333 414
666
221
666
